engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -2402,6 +2401,7 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
           LOG.warn("Cannot flush command history");
         }
       }
+      yarnClient.stop();
```

Comment:
Well.. since the call is coming from closeable in YarnAppHandlerImpl, calling close seemed more appropriate in that case. It's one of those weird situations, where close seems appropriate in this case (maybe in future there will be different handling when shutting down from a try block, with exception handling for example) and stop in the other. If you feel strongly that they should be consistent, I will change it to stop.